### ISSUE TITLE
Ability to add different scheduler address to workers outside of standard format

### ISSUE THREAD

**Issue by s-ducks** [dask/dask-kubernetes#842]
for some reason our k8s clusters don't use the `cluster.local` dns suffix and we would like to be able to add our own dns suffix to allow us to work around this. 
I have tried setting this with the env var `DASK_SCHEDULER_ADDRESS`, but this [bug](https://github.com/dask/dask-kubernetes/issues/841) prevents that from working

**Comment by jacobtomlinson** [comment]
I think this is a duplicate of #836  which will be closed by #837

**Comment by s-ducks** [comment]
yup agreed - have put a PR in: https://github.com/dask/dask-kubernetes/pull/843
lmk what you think :)

**Comment by wmay** [comment]
I just tried this with the most recent versions of the helm chart and dask kubernetes, and can confirm that it works now.
